### ISSUE TITLE
if __name__ ==  "__main__" needs to be explained some where.

### ISSUE THREAD

**Issue by deavmi** [WaspRC/Python-for-humans#5]
As it is not in here: https://github.com/WaspRC/Python-for-humans/blob/master/your-first-program.md

**Comment by Walkman100** [comment]
It's explained enough there for now, we can add it later in more advanced parts

**Comment by deavmi** [comment]
@Walkman100 That sounds like a good point. Maybe in a **"Writing quality programs"** section.

**Comment by deavmi** [comment]
> "It's explained enough there for now"

True, the book has to be read from beginning to ending and hence we need not repeat ourselves in each chapter with the same information (which Is something I do quite a lot) however I have removed some of that in the **"Tuples"** chapter which follows on from the **"Lists"** chapter and they are similiar structures in Python in the way they are represented and used.
